Directory.Build.props

```diff
@@ -48,7 +48,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
     <PackageProjectUrl>https://github.com/Microsoft/dotnet-apiport</PackageProjectUrl>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
```

Comment:
FWIW, we mostly set it to false these days. It's just `true` happens to be the default, so we're not entirely consistent. I'd rather we change try-convert 😄

However, for global tools it doesn't really matter `dotnet tool install` seems to ignore it.

Reply:
I see, in that case, I would rather set it back to false to avoid potential prompt from any system.